packages/types/src/sth-rest-api/data-transfer-objects/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./sequence-dto";
```

Comment:
I'm not especially fond of this sort of magic exports.

Reply:
removed